composer.json

```diff
@@ -5,9 +5,9 @@
     "description": "CrossSelling plugin for Sylius applications.",
     "license": "MIT",
     "require": {
-        "php": "^7.3 || ^7.4 || ^8.0",
+        "php": "^8.0",
         "bitbag/coding-standard": "^1.0",
-        "friendsofsymfony/elastica-bundle": "^5.2",
+        "friendsofsymfony/elastica-bundle": "^6.0",
```

Comment:
Why not ^6.1? I think better would be to update dependencies into newest version if possible